ui/static/ui/js/learning_resources.jsx

```diff
@@ -294,6 +295,32 @@ define('learning_resources',
         />;
       }
 
+      actionButtons = <p>
+        <button className="btn btn-lg btn-primary"
+                onClick={this.saveLearningResourcePanel}>
+          Save
+        </button>  <button className="btn btn-lg btn-success"
+                onClick={this.saveAndCloseLearningResourcePanel}>
+          Save and Close
+        </button>
+      </p>;
+      if (!_.isEmpty(this.state.message) &&
+        !_.isEmpty(this.state.message.error)) {
+        // show cancel button if error message
+        actionButtons = <p>
```

Comment:
nit: Is there a way to remove the duplication (lines 299-305 and lines 311-317)?